content/get-involved.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Get Involved
+---
+
+Jaeger is an open source project with [open governance](https://github.com/jaegertracing/jaeger/blob/master/GOVERNANCE.md). We welcome contributions from the community, and we’d love your help to improve and extend the project. Here are some ideas for how to get involved with the project. Some of them don't even require any coding.
+
+## No-coding involvement
+
+* Join the [online chat room](../get-in-touch) and help answering questions from the rest of the community.
```

Comment:
Create feature requests suggesting new capabilities to improve Jaeger?

Reply:
OK, but I'll rephrase to "propose new designs", since just requesting features is not particularly helpful to the project, it only grows the backlog.